README.md

```diff
@@ -35,7 +35,7 @@ To run a Docker container with [Work Directory](https://github.com/jenkinsci/rem
     
   Windows agent:
 
-    docker run jenkins/jnlp-agent-windows -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
+    docker run jenkins/inbound-agent:latest-windows -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
```

Comment:
I think we can just say jenkins/inbound-agent:latest here

Reply:
No, it is an example for Windows

Reply:
OK, now I see what's the problem. We have a naming collision between `latest` in the baseline and `latest` in Windows scripts. So we will get into https://github.com/jenkinsci/docker-agent/pull/126 equivalent if it is merged as is

```suggestion
    docker run jenkins/inbound-agent:latest-windows -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
```

Reply:
I don't think we are pushing a latest-windows right now. We probably want to use something else here:
```suggestion
    docker run jenkins/inbound-agent:windowsservercore-1809 -Url http://jenkins-server:port -WorkDir=C:/Jenkins/agent -Secret <secret> -Name <agent name>
```